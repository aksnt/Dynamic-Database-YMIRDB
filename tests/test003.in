SET a 1 2 3
SNAPSHOT
SNAPSHOT
SNAPSHOT
DROP 3
DROP 2
LIST SNAPSHOTS
BYE
