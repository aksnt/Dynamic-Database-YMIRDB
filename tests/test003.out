> ok

> saved as snapshot 1

> saved as snapshot 2

> saved as snapshot 3

> ok

> 2
1
